kythe/proto/storage.proto

```diff
@@ -203,3 +204,10 @@ message VNameRewriteRule {
 message VNameRewriteRules {
   repeated VNameRewriteRule rule = 1;
 }
+
+// RepoMetadata provides storage for additional information about a specific
+// build from a repository. This message is stored in the
+// CompilationUnit.details field.
+message BuildMetadata {
```

Comment:
Maybe this belongs in analysis.proto alongside CompilationUnit? Or maybe buildinfo.proto?

Reply:
Moved to analysis.proto.